modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -34,4 +49,83 @@ export default class NestedAccordion extends Base {
 			$items[ index ].appendChild( element );
 		} );
 	}
+
+	bindEvents() {
+		setTimeout( this.bindAnimationListeners.bind( this ), 200 ); // Wait for content to load before binding events.
+	}
+
+	unbindEvents() {
+		this.removeAnimationListeners();
+	}
+
+	bindAnimationListeners() {
+		const { $titles, $items, $accordionContent } = this.getDefaultElements();
+
+		$titles.each( ( index, title ) => {
+			const clickListener = this.clickListener.bind( this, $items, $accordionContent, index, title );
+			title.addEventListener( 'click', clickListener );
+			title.clickListener = clickListener; // Save reference for later removal
+		} );
+	}
+
+	clickListener( $items, $accordionContent, index, title, e ) {
+		e.preventDefault();
+
+		const item = $items[ index ],
+			content = $accordionContent[ index ];
+
+		if ( ! item.open ) {
+			this.open( item, title, content );
+		} else {
+			this.shrink( item, title );
+		}
+	}
+
+	animateItem( item, startHeight, endHeight, isOpen ) {
+		item.style.overflow = 'hidden';
+		let animation = this.animations.get( item );
+
+		if ( animation ) {
+			animation.cancel();
+		}
+
+		animation = item.animate( { height: [ startHeight, endHeight ] }, { duration: ANIMATION_DURATION } );
+		animation.onfinish = () => this.onAnimationFinish( item, isOpen );
+		this.animations.set( item, animation );
+	}
+
+	shrink( item, itemTitle ) {
```

Comment:
Could you use a more descriptibe function name, which explains what is done inside the function?